### PR TITLE
Changed target of TypeScript compilation to es2016

### PR DESCRIPTION
**Pull request by timocov**
**Type of PR:** enhancement

**PR checklist:**

- [x] Addresses an existing issue: partially addressed to #745
- [ ] Includes tests
- [x] Documentation update

### CODE REVIEW

**Review by kirchet — APPROVED** [review]

File: website/docs/intro.md [hunk 1]
```diff
@@ -6,6 +6,15 @@ sidebar_position: 0
 
 # Getting started
 
+## Requirements
+
+First of all, Lightweight Charts is _a client-side_ library.
+This means that it does not and cannot work on the server-side (i.e. NodeJS), at least out of the box.
+
+The code of `lightweight-charts` package is targeted to _es2016_ language specification.
```
Comment (romfrancois) [inline website/docs/intro.md:14]: Is it worth adding a link to those specs?

File: website/docs/intro.md [hunk 2]
```diff
@@ -6,6 +6,15 @@ sidebar_position: 0
...
+Thus, all the browsers you need to work with should support this language revision (see [this compatibility table](https://kangax.github.io/compat-table/es2016plus/)).
```
Comment (romfrancois) [inline website/docs/intro.md:15]: I would rephrase it as:
"Thus, [all] browsers should support ..."

File: website/docs/intro.md [hunk 3]
```diff
@@ -6,6 +6,15 @@ sidebar_position: 0
...
+If you need to support the previous revisions, you could try to setup a transpilation of the package to the target you need to support in your build system (e.g. by using Babel).
```
Comment (romfrancois) [inline website/docs/intro.md:16]: I would add a warning/info to mention we won't provide any or very limited help for that!

File: website/docs/intro.md (hunk 1 above)
Comment (timocov) [reply website/docs/intro.md:14]: 5d49c8c0de4514b3ee10d6ac5d852ea410c8197c

File: website/docs/intro.md (hunk 2 above)
Comment (timocov) [reply website/docs/intro.md:15]: 5d49c8c0de4514b3ee10d6ac5d852ea410c8197c

File: website/docs/intro.md (hunk 3 above)
Comment (timocov) [reply website/docs/intro.md:16]: 5d49c8c0de4514b3ee10d6ac5d852ea410c8197c
